Makefile

```diff
@@ -90,13 +104,16 @@ assets-clean:
 assets: repo/assets
 	mkdir -p $(testdir)/assets
 	cp -r -t $(testdir)/assets repo/assets/data/*
-	mkdir -p models
-	make model
-	cp -r  models/ $(testdir)/assets/dfki-testdata/data/
```

Comment:
This step used to place the downloaded models right into the workspaces of the CLI tests. You have configured your new unit test for dewarping to use the download location directly. But what about the others, won't this break them?

Until we have a relative path name resource resolution in core, maybe we should continue this (bad) practice.

Reply:
The main reason was to reduce disk I/O while testing. Waiting for ~1 GB data to be copied is a slight but annoying time waster while just interested in the unit test. I'll re-add them.